### ISSUE TITLE
Enable creating Razor Component class libraries in VS

### ISSUE THREAD

**Issue by danroth27** [aspnet/AspNetCore#5513]
Currently we have a Blazor Class Library for creating component libraries and JS interop libraries, but it's only available from the command-line. The issue tracks enabling a project template experience in VS for creating component class libraries.

**Comment by danroth27** [comment]
Proposal is to update the existing Razor Class Library template. By default we want Razor Class Library to support components for both client and server. But it will have an option for supporting pages & views, in which case it will then target server only.

**Comment by javiercn** [comment]
The templating work is done. Except for flipping the default value.
`dotnet new razorclasslib` produces the same traditional Razor Class Library.
`dotnet new razorclasslib -s false` produces a Razor Class Library without views and pages support (A Components Class Library).

Once the tooling work is done, we can flip the default to be true so
`dotnet new razorclasslib` produces a Razor Class Library without views and pages support (A Components Class Library).
`dotnet new razorclasslib -s` produces the same traditional Razor Class Library

Assigning do @danroth27 for tracking the required tooling work. Once that's done notify me and I'll update the default for the CLI experience.

**Comment by danroth27** [comment]
> The templating work is done. Except for flipping the default value.

I'll open a separate issue to track this.
